### PR TITLE
add quick search select box to home page

### PR DESCRIPTION
**Pull request by stevenpmcfarlane**
# Description
Added a quick search select box for 10 days and 30 days to the home page. Works on the dropdown menu, not the map. Needs styling
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ X] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [X ] My code follows the style guidelines of this project
- [X ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X ] There are no merge conflicts

### CODE REVIEW

**Comment by robert-petersen** [comment]
Looks great so far. I think it's a really good filtering addition for looking at the data.

File: src/components/Home/Map/Search.js [hunk 1]
```diff
@@ -87,17 +97,27 @@ export default function Search({ zoomOnCluster }) {
...
       <RangePicker
         style={{ width: '100%' }}
         disabledDate={disableFutureDates}
         onCalendarChange={handleCalendarChange}
       />
+      <div>
+        <label>
+          Quick Search
+          <select onChange={onChange} value={quickSelect} name="quicksearch">
+            <option value="">- Select an option -</option>
```
Comment (robert-petersen) [inline src/components/Home/Map/Search.js:115]: Maybe have the default option be something like "No time filter" or "show all data" (neither of those sounds too great to me but you get the point) so the user knows they don't have to select a time frame filter.

**Review by robert-petersen — COMMENTED** [review]
Like I said before (but wanted to leave a review instead of a comment this time) this looks good so far. I think we should work on the styling a bit more before merging to the main branch but I really like this idea.

**Review by john-chamberlin — APPROVED** [review]

**Review by Brandonx1123 — COMMENTED** [review]
i think it is fine with the codebase, but we need to make sure it is styled and located where it needs to be on the map

**Review by Brandonx1123 — APPROVED** [review]

**Pull request merged**